index.html

```diff
@@ -1,19 +1,35 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Web-design</title>
-    <link rel="stylesheet" href="css/sheet.css">
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Web-design</title>
+  <link rel="stylesheet" href="css/sheet.css">
 </head>
 <body>
-   <header >
-       <nav>
-            <div class="bg-black"><img src="image/LOGO_RW.png"></div>
-       </nav>
-      
-   </header>
+  <header>
+    <nav>
+      <div class="bg-black"><img src="image/LOGO_RW.png"></div>
+    </nav>
+    <div>
+    </div>
+  </header>
+  <section>
+    <div class="container">
+     <div class="text-white">
```

Comment:
user `.row `and `.col-50` to define 2 divs that occupy 50% of width. And add content inside them